server/controllers/authController.js

```diff
@@ -12,6 +12,8 @@ const signUp = async (req, res) => {
         email
     });
 
+    delete user.password;
```

Comment:
Hello. Let's handle this differently. Create a method called `response` on the user model. This method will generate a response we can use anytime we want to send user details. Here's an example:

```js
/**
     *
     * Get a customer response object for the frontend
     *
     * @return {Promise}
     *
     */
    async response() {
        return {
            user: {
                    name: this.name,
                    email: this.email,
            }
            token: await this.getToken()
        }
    }
```

And now in the controller, we can simply do :
```js
res.status(201).jsend(await user.response())
```